netneurotools/plotting.py

```diff
@@ -286,8 +286,8 @@ def plot_conte69(data, lhlabel, rhlabel, surf='midthickness',
     return lhplot, rhplot
 
 
-def plot_fsaverage(data, lhannot, rhannot, *, surf='pial', views='lat',
-                   vmin=None, vmax=None, center=None, mask=None,
+def plot_fsaverage(data, lhannot, rhannot, *, order='LR', surf='pial',
```

Comment:
```suggestion
def plot_fsaverage(data, *, lhannot, rhannot, order='LR', surf='pial',
```

We talked about also making the `lhannot` and `rhannot` arguments required keyword arguments; moving the `*` before them in the function definition will do this. Thus, people will have to specify:

```python
plot_fsaverage(data, lhannot='/path/to/lh.annot', rhannot='/path/to/rh.annot')
```

with the `lhannot` and `rhannot` keys. This would prevent accidentally flipping them when providing the files as positional arguments.